apps/next/src/pages/AccountManagement/components/AddOrCreateWallet/AddOrConnectWallet.tsx

```diff
@@ -22,11 +22,18 @@ const underDevelopmentClick = () => toast.error('Under development');
 
 export const AddOrConnectWallet: FC = () => {
   const { t } = useTranslation();
+  const { capture } = useAnalyticsContext();
+
   const { addAccount, accounts, selectAccount } = useAccountsContext();
-  const { goBack } = useHistory();
+  const { goBack, push } = useHistory();
 
   const isPrimaryAccount = accounts.active?.type === AccountType.PRIMARY;
 
+  const goToImportPrivateKey = useCallback(() => {
+    capture('ImportPrivateKey_Clicked');
```

Comment:
For the other analytics call there is no `_` in the key. I'm not sure whether this matters or not 😅 
\#justAsking

Reply:
talked offline, this is how it was used in the legacy app as well:)